llvmcpy/llvm.py

```diff
@@ -360,7 +362,9 @@ def iter_{}s(self):
                                                       name[4:]))
     return result
 
-header_blacklist = ["llvm/Support/DataTypes.h",
+header_blacklist = ["inttypes.h",
+                    "llvm/Support/DataTypes.h",
```

Comment:
My LLVM 8 installation includes DataTypes.h from llvm-c:

    #include "llvm-c/DataTypes.h"

So it currently gets missed. I think adding "llvm-c/DataTypes.h" will also prevent the need for "inttypes.h" because it's included by DataTypes.h,